app/models/article/node.rb

```diff
@@ -1,24 +1,25 @@
-module Article::Node
-  class Base
-    include Cms::Model::Node
-
-    default_scope ->{ where(route: /^article\//) }
-  end
-
-  class Page
-    include Cms::Model::Node
-    include Cms::Addon::NodeSetting
-    include Cms::Addon::Meta
-    include Event::Addon::PageList
-    include Category::Addon::Setting
-    include Cms::Addon::TagSetting
-    include Cms::Addon::OpendataRef::Site
-    include Cms::Addon::Release
-    include Cms::Addon::DefaultReleasePlan
-    include Cms::Addon::MaxFileSizeSetting
-    include Cms::Addon::GroupPermission
-    include History::Addon::Backup
-
-    default_scope ->{ where(route: "article/page") }
-  end
-end
+module Article::Node
+  class Base
+    include Cms::Model::Node
+
+    default_scope ->{ where(route: /^article\//) }
+  end
+
+  class Page
+    include Cms::Model::Node
+    include Cms::Addon::NodeSetting
+    include Cms::Addon::Meta
+    include Cms::Addon::NodeAutoPostSetting
+    include Event::Addon::PageList
+    include Category::Addon::Setting
+    include Cms::Addon::TagSetting
+    include Cms::Addon::OpendataRef::Site
+    include Cms::Addon::Release
+    include Cms::Addon::DefaultReleasePlan
+    include Cms::Addon::MaxFileSizeSetting
+    include Cms::Addon::GroupPermission
+    include History::Addon::Backup
+
+    default_scope ->{ where(route: "article/page") }

```

Comment:
__[rails_best_practices]__
* default_scope is evil [:ref](http://rails-bestpractices.com/posts/2013/06/15/default_scope-is-evil/)

<!-- Comment by [SideCI](https://sideci.com) -->